recipes/hipace/bld.bat

```diff
@@ -0,0 +1,45 @@
+@echo on
+
+:: simple install prep
+::   copy all hipace*.exe and hipace*.dll files
+if not exist %LIBRARY_PREFIX%\bin md %LIBRARY_PREFIX%\bin
+if errorlevel 1 exit 1
+
+:: configure
+cmake ^
```

Comment:
Please add the BUILD_SHARED_LIBS CMake variable. We do not allow shipping static libraries.